aiida_quantumespresso/calculations/pp.py

```diff
@@ -197,6 +209,17 @@ def prepare_for_submission(self, folder):  # pylint: disable=too-many-branches,t
                 parent_calc_folder.computer.uuid,
                 os.path.join(parent_calc_folder.get_remote_path(), self._PSEUDO_SUBFOLDER), self._PSEUDO_SUBFOLDER
             ))
+
+            if copy_files:
+                remote_symlink_list.append((
+                    parent_calc_folder.computer.uuid,
+                    os.path.join(parent_calc_folder.get_remote_path(), f'aiida.in'), 'aiida.ref'
+                ))
+                remote_symlink_list.append((
+                    parent_calc_folder.computer.uuid,
+                    os.path.join(parent_calc_folder.get_remote_path(), f'{self._FILPLOT}_*'), '.'
+                ))
```

Comment:
I don't want to duplicate these files as they are tipically heavy, so just simlink them.